public/main.css

```diff
@@ -34,6 +38,393 @@
+  transform: translate(-38.5%, -85.5%);
+  border-radius: 3px;
+  width: 87%;
+  height: 72vh;
+  background-color: white;
+}
+
+#camera.wipe.finished #camera_glyph_icon_1 {
+  fill: #FDACAB;
+}
+
+.menu-open:checked ~ #microphone.wipe {
+  animation: microphone 1s ease-in-out;
+  -webkit-animation: microphone 1s ease-in-out;
+}
+
+#microphone.wipe #audio_outline_icon_2 {
+  animation: color-svg 1s ease-in-out;
+  -webkit-animation: color-svg 1s ease-in-out;
+}
+
+.menu-open:checked ~ #microphone.wipe.finished {
+  -webkit-transform: translate(-38.5%, -85.5%);
+  transform: translate(-38.5%, -85.5%);
+  border-radius: 3px;
+  width: 87%;
+  height: 72vh;
+  background-color: white;
+}
+
+#microphone.wipe.finished #audio_outline_icon_2 {
+  fill: #FDACAB;
+}
+
+.menu-open:checked ~ #clock.wipe {
+  animation: clock 1s ease-in-out;
+    -webkit-animation: clock 1s ease-in-out;
+}
+
+#clock.wipe #notification_glyph_icon_1 {
+  animation: color-svg 1s ease-in-out;
+  -webkit-animation: color-svg 1s ease-in-out;
+}
+
+.menu-open:checked ~ #clock.wipe.finished {
+  -webkit-transform: translate(-38.5%, -85.5%);
+  transform: translate(-38.5%, -85.5%);
+  border-radius: 3px;
+  width: 87%;
+  height: 72vh;
+  background-color: white;
+}
+
+#clock.wipe.finished #notification_glyph_icon_1 {
+  fill: #FDACAB;
+}
+
+.menu-open:checked ~ #cog.wipe {
+  animation: cog 1s ease-in-out;
+  -webkit-animation: cog 1s ease-in-out;
+}
+
+#cog.wipe #settings_glyph_icon_1 {
+  animation: color-svg 1s ease-in-out;
+  -webkit-animation: color-svg 1s ease-in-out;
+}
+
+.menu-open:checked ~ #cog.wipe.finished {
+  -webkit-transform: translate(-38.5%, -85.5%);
+  transform: translate(-38.5%, -85.5%);
+  border-radius: 3px;
+  width: 87%;
+  height: 72vh;
+  background-color: white;
+}
+
+#cog.wipe.finished #settings_glyph_icon_1 {
+  fill: #FDACAB;
+}
+
 .memory {
   transition: r 2s ease-in-out;
 }
+
+.tags {
+  position: absolute;
+    left: 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    cursor: pointer;
+    top: 30%;
+ }
+
+ .tag-container {
+   text-align: center;
+   height: 50%;
+     display: none;
+     justify-content: space-around;
+   list-style-type: none;
+   background: rgb(142,165,206);
+   border-radius: 30%;
+   align-items: center;
+ }
+
+
+ .tags li {
+      list-style-type: none;
+      display: none;
+      text-align: center;
+      height: 50%;
+ }
+
+.top-bar {
+  position: absolute;
+  z-index: 99999;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.shuffle-memories {
+  padding-left: 3%;
+  padding-top: 2%;
+}
+
+.search-tags {
+  padding-right: 3%;
+  padding-top: 2%;
+  transition: width 1s ease;
+}
+
+#to-extend {
+  transform-origin: bottom;
+  transition: transform 0.6s ease;
+}
+
+
+.underline {
+  width:90%;
+  height: 90%;
+    display: inline-block;
+    position: absolute;
+    z-index: -1;
+  right: 5%;
+
+}
+
+.underline:after {
```

Comment:
It refers to a line that extends underneath the tags - would there be a better name for it?